scipy/spatial/_spherical_voronoi.py

```diff
@@ -228,7 +228,7 @@ def __init__(self, points, radius=None, center=None, threshold=1e-06):
         else:
             self.radius = 1
 
-        if pdist(self.points).min() <= threshold * self.radius:
+        if len(cKDTree(self.points).query_pairs(threshold * self.radius)) > 0:
```

Comment:
The only other thought I had here is if you could even omit the `len > 0`

Reply:
Thanks for the suggestion - I have made the change.  By the way, are you able to squash the commits during merging?